data/newsletter/issue-380.markdown

```diff
@@ -0,0 +1,80 @@
+Welcome to another issue of Haskell Weekly!
+[Haskell](https://www.haskell.org) is a safe, purely functional programming language with a fast, concurrent runtime.
+This is a weekly summary of what's going on in its community.
+
+## Featured
+
+- [Anti-Instances in Haskell](https://www.heneli.dev/blog/anti-instances) by Heneli Kailahi
+> Typeclass instances are a powerful tool for specifying which overloaded operations are allowed on which types. But what about specifying which operations are disallowed on which types?
+
+- [Community Survey: Removing cabal’s nix integration](https://discourse.haskell.org/t/community-survey-removing-cabals-nix-integration/7201) by Hécate Moonlight
+> The Cabal team would like to ask our users if anyone would object to the Nix Integration 19 being removed (following a reasonable deprecation cycle).
+
+- [falsify: Internal Shrinking Reimagined for Haskell](http://www.edsko.net/pubs/falsify.pdf) (pdf) by Edsko de Vries
+> Randomly generated inputs tend to be large and should therefore be minimised. Traditionally this is done with an explicitly provided shrinker, but in this paper we propose a way to write generators that obsoletes the need to write a separate shrinker.
+
+- [Feedback wanted: Module naming conventions for GHC base libraries](https://github.com/haskellfoundation/tech-proposals/pull/53) 
+> The GHC team would like input on module/package naming for the planned restructuring of the libraries shipped with GHC. Please take a look at the proposal and let us know what you think!
+
+- [GHC 9.4.6 is now available](https://discourse.haskell.org/t/ghc-9-4-6-is-now-available/7231) by Ben Gamari
+> The GHC developers are happy to announce the availability of GHC 9.4.6.
+
+- [Reducing Haskell parallel build times using semaphores](https://well-typed.com/blog/2023/08/reducing-haskell-parallel-build-times/) by Well-Typed
+> This blog post explains a new feature for GHC & Cabal, which allows multiple simultaneously running GHC processes to coordinate their use of compute resources.
+
+- [Speeding up tree-sitter-haskell 50x](https://owen.cafe/posts/tree-sitter-haskell-perf/) by Owen Shepherd
+> This article is about profiling and picking the low-hanging fruit of tree-sitter-haskell (warning: C(++)).
+
+## Jobs
+
+undefined
```

Comment:
😜 

I was talking with Evgeny about running an ad for SimpleX Chat. I was going to add it this morning, but this PR had already been merged! I added it anyway: b8de2d8cee24d23b50118a3df72881f187cd8535. Still before the emails go out, so all is good.

Reply:
Oh, my bad. Should I hold off merging the PR until this section is filled out next time?

Reply:
Nah, I'll be sure to let you know when an ad is possible.